.circleci/config.yml

```diff
@@ -276,6 +285,20 @@ jobs:
           root: ~/
           paths: project/bin
 
+  cudareceiver:
+    executor:
+      name: cuda
+    steps:
+      - restore_workspace
+      - run:
+          name: Unit test for CUDA receiver
+          command: |
+            cd receiver/cudareceiver
+            make do-unit-tests-with-cover
```

Comment:
Do we also want to run lint commands or anything else? Since we are running this command from within the cudareceiver folder, I think you could create your own `.golangci.yml` config in that directory that excludes the problematic linters (staticcheck?)